spec/index.md

```diff
@@ -1596,6 +1594,8 @@ There must be at most one call to `ic0.call_on_cleanup` between `ic0.call_new` a
 
     This concludes assembling the call. It queues the call message to the given destination, but does not actually act on it until the current WebAssembly function returns without trapping.
 
+    This deducts `MAX_CYCLES_PER_RESPONSE` cycles from the canister balance and sets them aside for response processing. This will trap if not sufficient cycles are available.
```

Comment:
```suggestion
    This deducts `MAX_CYCLES_PER_RESPONSE` cycles from the canister balance and sets them aside for response processing.
```

Reply:
This is not happening in the implementation and I think it's also wrong to do that. If we can't make the reservation we should return a non-zero error code to signal that there were not enough cycles available.

Reply:
Good point! I've fixed that now (also in the formal specs).